### PR TITLE
Fixed an issue where user was not able to cancel or terminate active queries from dashboard. #7078

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/dashboard/static/js/Dashboard.jsx [hunk 1]
```diff
@@ -695,6 +695,7 @@ function Dashboard({
         ? treeNodeInfo.server.user.can_signal_backend
         : false;
 
+    let active_sessions = dashData.filter((data)=>data.state === 'active');
```
Comment (adityatoshniwal) [inline web/pgadmin/dashboard/static/js/Dashboard.jsx:698]: shouldn't this filter also include database name comparison with maintenance_database?

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/dashboard/static/js/Dashboard.jsx (hunk 1 above)
Comment (pravesh-sharma) [reply web/pgadmin/dashboard/static/js/Dashboard.jsx:698]: I have copied this logic from the old code. If we put a filter for maintenance_database, we will not be able to terminate/cancel the query running on the query tool of the maintenance database. Also, we can not terminate the connection made with the maintenance database (default connection), the backend will throw the error which is handled.

Comment (adityatoshniwal) [reply web/pgadmin/dashboard/static/js/Dashboard.jsx:698]: as per the logic, it is trying to check if there are more than one connection for the connection being terminated. But what if there are multiple DB connections?

Comment (pravesh-sharma) [reply web/pgadmin/dashboard/static/js/Dashboard.jsx:698]: If you check line 732, the condition is if we have more than one active connection then we should be able to terminate.

Comment (adityatoshniwal) [reply web/pgadmin/dashboard/static/js/Dashboard.jsx:698]: What I want to ask is - if there are 2 active connections in the dashboard with both connected to different databases, will it allow terminating the maintenance one?

**Pull request merged**